app/controllers/home_controller.rb

```diff
@@ -8,4 +9,14 @@ def index
       :coordinator, :accommodation_type, :user
     )
   end
+
+  def show
+    @place
+  end
+
+  private
+
+  def set_place
+    @place = Place.find(params[:id])
```

Comment:
need to check whether provided place id belongs to available places for given user

Reply:
All places have specific id,  and show endpoint pass this id.

Reply:
when you pass in random id, access should be denied
we need to check, that passed id is available for certain user

Reply:
Added checking specific available place for current user 
Addressed here https://github.com/vova/war_shelter/pull/79/commits/559ff1a4c6a8f85021e0b4a2c720102ee30949eb